test/quantization/test_quantize_fx.py

```diff
@@ -1592,6 +1594,68 @@ def forward(self, x):
         assert hasattr(m, "mods1_1_packed_weight_0")
         assert hasattr(m, "mods2_packed_weight_0")
 
+    def test_state_dict(self):
```

Comment:
would it also make sense to test saving to disk and successfully loading back?

Reply:
Oh sure, yeah I think that would be useful